pkg/cmd/vault-plugin-secrets-omd-reader/main.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2022 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"os"
+
+	"github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/vault/api"
+	"github.com/hashicorp/vault/sdk/plugin"
+
+	omsecrets "fybrik.io/vault-plugin-secrets-omd-reader/pkg/omsecrets"
+)
+
+const ErrorParseArguments = 1
+const ErrorServerPlugin = 2
+
+func main() {
+	// Boilerplate code to get started.
+	// Please see https://www.hashicorp.com/blog/building-a-vault-secure-plugin for more info.
+	apiClientMeta := &api.PluginAPIClientMeta{}
+	flags := apiClientMeta.FlagSet()
+	err := flags.Parse(os.Args[1:]) //nolint
```

Comment:
we don't need the `nolint` directive
```
pkg/cmd/vault-plugin-secrets-omd-reader/main.go:24:34: directive `//nolint` is unused (nolintlint)
	err := flags.Parse(os.Args[1:]) //nolint
```

Reply:
Since I upgraded the golint docker image version to v3, I have to add this `//nolint` comment, or else I get the following error:
```
add-constant: avoid magic numbers like '1', create a named constant for it (revive)
```